tests/unit_tests/shared/test_feature_toggling.py

```diff
@@ -0,0 +1,69 @@
+import os
+from unittest.mock import Mock
+
+import pytest
+
+import ert.__main__
+from ert.__main__ import ert_parser
+from ert.cli import TEST_RUN_MODE
+from ert.shared.feature_toggling import FeatureToggling
+
+
+@pytest.fixture(autouse=True)
+def mocked_valid_file(monkeypatch):
+    monkeypatch.setattr(ert.__main__, "valid_file", Mock(return_value=""))
+
+
+def test_feature_toggling_from_env(monkeypatch):
+    envs = {"ERT_ENABLE_SCHEDULER": True}
+    monkeypatch.setattr(os, "environ", envs)
+
+    assert not FeatureToggling.is_enabled("scheduler")
+
+    FeatureToggling.update_from_environment()
```

Comment:
`FeatureToggling` is a singleton. How are you ensuring that the tests don't leak?

Hint: Use a `pytest.fixture` with `autouse=True` in this module that sets the values to some default.

Reply:
this was fixed